test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/RuntimeTypeInfoTest.cs

```diff
@@ -13,44 +13,18 @@ namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
 {
     public class RuntimeTypeInfoTest
     {
-        private static readonly string StringFullName = typeof(string).FullName;
-        private static readonly string CollectionsNamespace = typeof(IDictionary<,>).Namespace;
-
-        public static TheoryData RuntimeTypeInfo_ReturnsMetadataOfAdaptingTypeData =>
-            new TheoryData<Type, string>
-            {
-                { typeof(int), typeof(int).FullName },
-                { typeof(string), typeof(string).FullName },
-                { typeof(Tuple<>), typeof(Tuple<>).FullName },
-                { typeof(Tuple<,>), typeof(Tuple<,>).FullName },
-                {
-                    typeof(IDictionary<string, string>),
-                    $"{typeof(IDictionary<,>).FullName}[[{StringFullName}],[{StringFullName}]]"
-                },
-                {
-                    typeof(IDictionary<string, IDictionary<string, CustomType>>),
-                    $"{typeof(IDictionary<,>).FullName}[[{StringFullName}],[{typeof(IDictionary<,>).FullName}" +
-                    $"[[{StringFullName}],[{typeof(CustomType).FullName}]]]]"
-                },
-                {
-                    typeof(IList<IReadOnlyList<IDictionary<List<string>, Tuple<CustomType, object[]>>>>),
-                    $"{typeof(IList<>).FullName}[[{typeof(IReadOnlyList<>).FullName}[[{typeof(IDictionary<,>).FullName}[[" +
-                    $"{typeof(List<>).FullName}[[{StringFullName}]]],[{typeof(Tuple<,>).FullName}[[{typeof(CustomType).FullName}]," +
-                    $"[{typeof(object).FullName}[]]]]]]]]]"
-                },
-                { typeof(AbstractType), typeof(AbstractType).FullName },
-                { typeof(PrivateType), typeof(PrivateType).FullName },
-                { typeof(KnownKeyDictionary<>), typeof(KnownKeyDictionary<>).FullName },
-                {
-                    typeof(KnownKeyDictionary<string>),
-                    $"{typeof(KnownKeyDictionary<>).Namespace}" +
-                    $".RuntimeTypeInfoTest+KnownKeyDictionary`1[[{StringFullName}]]"
-                }
-            };
-
         [Theory]
-        [MemberData(nameof(RuntimeTypeInfo_ReturnsMetadataOfAdaptingTypeData))]
-        public void RuntimeTypeInfo_ReturnsMetadataOfAdaptingType(Type type, string expectedFullName)
+        [InlineData(typeof(int))]
+        [InlineData(typeof(string))]
+        [InlineData(typeof(Tuple<>))]
+        [InlineData(typeof(Tuple<>))]
+        [InlineData(typeof(IDictionary<string, string>))]
```

Comment:
Most of those cases aren't super interesting per se now - they were more useful when we were sanitizing the `FullName`s